src/Tests/dotnet-MsiInstallation.Tests/VSWorkloadTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.DotNet.MsiInstallerTests.Framework;
+
+namespace Microsoft.DotNet.MsiInstallerTests
+{
+    public class VSWorkloadTests : VMTestBase
+    {
+        public VSWorkloadTests(ITestOutputHelper log) : base(log)
+        {
+            VM.SetCurrentState("Install VS 17.10 Preview 6");
```

Comment:
What installs this build of VS? This is just the name of the snapshot, correct?

Reply:
Yes, this is part of the test setup, you have to install VS and create a snapshot with the right name.